src/lumigo_tracer/lumigo_utils.py

```diff
@@ -305,6 +312,9 @@ def report_json(region: Optional[str], msgs: List[dict], should_retry: bool = Tr
     except Exception as e:
         get_logger().exception("Failed to create request: A span was lost.", exc_info=e)
         return 0
+    if should_use_tracer_extension():
+        write_spans_to_file(to_send)
+        return 0
```

Comment:
This function suppose to return the duration of the reporting. Writing to file still takes some time, don't we want to measure it?